rho/cli.py

```diff
@@ -112,9 +112,14 @@ def main(self):
         to find the best command match. If no match is found the
         usage is displayed
         """
-        if len(sys.argv) < 2 or not self._find_best_match(sys.argv):
-            self._usage()
-            sys.exit(1)
+        len_sys_argv = len(sys.argv)
```

Comment:
made this a variable because we'll use this check when we add the `rho --version` command next sprint.